index.js

```diff
@@ -244,106 +242,85 @@ const getSettings = function(options) {
  *         they need to be exported to rollup as an Array.
  */
 const generateRollupConfig = function(options) {
-  options = options || {};
   const settings = getSettings(options);
 
   /* make a build with the specifed settings */
   const makeBuild = (buildType, buildOverrides, buildSettings = settings) => {
-    const b = Object.assign({}, {
-      plugins: buildSettings.plugins[buildType],
+    const b = Object.assign({
+      // never clear screen during watch
+      watch: {clearScreen: false},
+      // map plugin names from strings to primed plugins
+      // but only if the plugin is a string and not a
+      // primed plugin already.
+      plugins: buildSettings.plugins[buildType].map((p) => typeof p !== 'string' ? p : buildSettings.primedPlugins[p]),
       external: buildSettings.externals[buildType],
-      input: buildSettings.input
+      input: buildType === 'test' ? buildSettings.testInput : buildSettings.input
```

Comment:
Previously we specified test input elsewhere, but I think that it makes more sense to do it here.